### PR TITLE
updated badrequest and gone returns for rfc 7807 compliance

### PR DESCRIPTION
**Pull request by kirbyAlasdairAnswer**

### CODE REVIEW

File: src/Answer.King.Api/Program.cs [hunk 1]
```diff
@@ -50,7 +51,10 @@
 });
 
 builder.Services.AddValidatorsFromAssemblyContaining<Program>();
-builder.Services.AddFluentValidationAutoValidation();
+builder.Services.AddFluentValidationAutoValidation(fv =>
```
Comment (milkshakeuk) [inline src/Answer.King.Api/Program.cs]: sinced `fv` is not used you might want to change it to an underscore `_`

**Review by milkshakeuk — CHANGES_REQUESTED** [review]
See comment.

**Review by mbaltrimas — DISMISSED** [review]

File: src/Answer.King.Api/Program.cs [hunk 2]
```diff
@@ -50,7 +51,10 @@
 });
 
 builder.Services.AddValidatorsFromAssemblyContaining<Program>();
-builder.Services.AddFluentValidationAutoValidation();
+builder.Services.AddFluentValidationAutoValidation(_ =>
```
Comment (milkshakeuk) [inline src/Answer.King.Api/Program.cs]:
```csharp
builder.Services.AddFluentValidationAutoValidation(
    _ => ValidatorOptions.Global.PropertyNameResolver = CamelCasePropertyNameResolver.ResolvePropertyName);
```

**Review by milkshakeuk — CHANGES_REQUESTED** [review]
last change 😁

**Review by milkshakeuk — APPROVED** [review]

**Review by mbaltrimas — APPROVED** [review]

**Pull request merged**